src/navigators/PopupWindow.ts

```diff
@@ -57,12 +57,15 @@ export class PopupWindow extends AbstractChildWindow {
     public async navigate(params: NavigateParams): Promise<NavigateResponse> {
         this._window?.focus();
 
-        const popupClosedInterval = setInterval(() => {
+        const popupClosedInterval = { id: setInterval(() => {
             if (!this._window || this._window.closed) {
-                void this._abort.raise(new Error("Popup closed by user"));
+                this._logger.debug("Popup closed by user or isolated by redirect");
+                clearInterval(popupClosedInterval.id);
+                // appease TypeScript while not clearing twice
```

Comment:
why is this needed? Here TypeScript has no problem with this simpler code:
```ts
...
        const popupClosedInterval = setInterval(() => {
            if (!this._window || this._window.closed) {
                this._logger.debug("Popup closed by user or isolated by redirect");
                clearInterval(popupClosedInterval);
            }
        }, checkForPopupClosedInterval);
        this._disposeHandlers.add(() => clearInterval(popupClosedInterval));
...
```

Reply:
I'm trying to avoid clearing the same interval twice (I can't find anything in the standard that says that is legal). I've also been thinking about closures in C++ instead of JavaScript. Modified to also remove the dispose handler.

Reply:
Ah i understand. I have also not found something in the standard. Your code looks good enough. Multiple calls to clearInterval with same id seems to work fine...